server/libs/infra/src/entities/asset.entity.ts

```diff
@@ -51,6 +51,9 @@ export class AssetEntity {
   @Column({ type: 'varchar', nullable: true, default: '' })
   webpPath!: string | null;
 
+  @Column({ type: 'varchar', nullable: true, default: '' })
```

Comment:
this should be a `bytea` I believe. We can store raw bytes, similar to the checksum field.